### ISSUE TITLE
`goctl rpc proto ` failed for invalid import path 

### ISSUE THREAD

**Issue by youzipi** [zeromicro/go-zero#1436]
**Describe the bug**
follow this article(https://go-zero.dev/cn/shorturl.html),
use goctl to generate grpc code, throw error: `invalid import path "."` 

**To Reproduce**
Steps to reproduce the behavior, if applicable:

transform.ptoto:
```proto
syntax = "proto3";

package transform;


message expandReq {
  string shorten = 1;
}

message expandResp {
  string url = 1;
}

message shortenReq {
  string url = 1;
}

message shortenResp {
  string shorten = 1;
}

service Transform {
  rpc expand(expandReq) returns(expandResp);
  rpc shorten(shortenReq) returns(shortenResp);
}

```

cmd:
```shell
goctl rpc proto -src transform.proto -dir .

>
protoc  --proto_path=/Users/mac/projects/zaihui/obol/tinyurl transform.proto --go_out=plugins=grpc:/Users/mac/projects/zaihui/obol/tinyurl/transform --go_opt=Mtransform.proto=.
goctl: generation error: protoc-gen-go: invalid Go import path "." for "transform.proto"

The import path must contain at least one forward slash ('/') character.

See https://developers.google.com/protocol-buffers/docs/reference/go-generated#package for more information.

--go_out: protoc-gen-go: Plugin failed with status code 1.
goctl version: 1.2.5 darwin/amd64

   ```

**Expected behavior**

**Screenshots**

**Environments (please complete the following information):**
 - OS: macOS 10.15.7
 - go-zero version: 1.2.5
 - goctl version: 1.2.5
```
protoc --version
libprotoc 3.19.2

 protoc-gen-go --version
protoc-gen-go v1.26.0

```

**More description**

**Comment by anqiansong** [comment]
Use `goctl rpc protoc` instead, for detail see https://github.com/zeromicro/go-zero/pull/1251

**Comment by jsonMark** [comment]
goctl version 1.2.3-cli windows/amd64
D:\Tools\Go\Code\src\demo>protoc --version
libprotoc 3.19.1
github.com/tal-tech/go-zero v1.2.4
test ok

**Comment by qingliu-cyl** [comment]
add the code  `option go_package = "pb";` in proto file

**Comment by jsonMark** [comment]
error list :
1. add the code option go_package = "pb"; in proto file
2. Use goctl rpc protoc instead, for detail see https://github.com/zeromicro/go-zero/pull/1251
3. suggesst you update goctl of v1.3.0 and exec go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1
your protoc-gen-go version(1.26.0) is not fix goctl version: 1.2.5,full detail to see my doc :https://learnku.com/articles/65090
